<launch>

	<node pkg="node_edge_localizer" type="node_edge_localizer" name="node_edge_localizer" ns="localiztion">
			<param name="INIT_NODE0_ID" value="0" type="int"/>
			<param name="INIT_NODE1_ID" value="1" type="int"/>
			<param name="INIT_PROGRESS" value="0.0" type="double"/>
			<param name="INIT_YAW" value="0.0" type="double"/>
	</node>

</launch>
